packages/exeinfope.vm/exeinfope.vm.nuspec

```diff
@@ -2,12 +2,13 @@
 <package xmlns="http://schemas.microsoft.com/packaging/2015/06/nuspec.xsd">
   <metadata>
     <id>exeinfope.vm</id>
-    <version>0.0.7.20250219</version>
+    <version>0.0.7.20250505</version>
     <authors>A.S.L Soft</authors>
-    <description>Displays metadata for a variety of file types and identifies many executable packers</description>
+    <description>Exeinfo Pe displays metadata for a variety of file types and identifies many executable packers.</description>
```

Comment:
[nitpick] The author seems to use both `Exeinfo PE` and `Exeinfo Pe` in different places of the documentation, but the tool seem to be called `Exeinfo PE`:
![image](https://github.com/user-attachments/assets/bb7697f7-8370-4ae4-902f-7b15c6b9f3f0)

```suggestion
    <description>Exeinfo PE displays metadata for a variety of file types and identifies many executable packers.</description>
```